app/services/proofing/aamva/request/verification_request.rb

```diff
@@ -181,10 +221,22 @@ def add_optional_element(name, value:, document:, inside: nil, after: nil)
           end
         end
 
+        # @param document [REXML::Document]
+        def update_requested_attributes(document)
+          VERIFICATION_REQUESTED_ATTRS.each do |attribute, rule|
```

Comment:
This is good, I like that we are sourcing this from the actual XML we are building.